### PR TITLE
Awesome books: Refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by olaishola05**
# Refactor to use JavaScript classes

- Refactor code to use JavaScript classes instead of objects and arrays
- Create class methods to add and remove books.
- Add CSS styles to the application to make it match the wireframe given

### CODE REVIEW

**Comment by ahmedtaa** [comment]
Nice work .. let's submit the code for review

**Comment by olaishola05** [comment]
Good job guys, great teamwork and spirit from you guys :muscle:

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
-# awesome-books
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# A book collection app
+
+![awesome book app](./public/images/awesome.png)
```
Comment (moise10r) [inline README.md]: - [OPTIONAL ] Please remove this image, since you already have the updated one.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  border: 1px solid #000;
+  border-radius: 2px;
+  padding: 5px;
+  width: 300px;
+  height: 30px;
+}
+
+.form-container form .error {
+  visibility: hidden;
+  color: red;
```
Comment (moise10r) [inline style.css]: - [OPTIONAL ] Geat job to implement the error message :clap: , Kindly follow the general layout, chance the error message color to match the design

**Review by moise10r — APPROVED** [review]
###  **Hi Team,** 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

Cheers and Happy coding!👏👏👏


 #### Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (olaishola05) [reply style.css]: Thanks, will adapt the error message to follow the general layout to follow the design color and remove the above-mentioned image from the readme file.

**Comment by olaishola05** [comment]
Done. :sunglasses:

**Pull request merged**